source/Buoy-Comparison-Tests/PropertyBasedEqualityCheckerTest.class.st

```diff
@@ -25,6 +25,22 @@ PropertyBasedEqualityCheckerTest >> testCheckingIdenticalObjects [
 	self assert: ( checker checkAgainst: self )
 ]
 
+{ #category : 'tests' }
+PropertyBasedEqualityCheckerTest >> testConfiguringTypeComparison [
+
+  | checker |
+  checker := PropertyBasedEqualityChecker
+               on: 8
+               withTypeChecking: [ :base :target | target isNumber ].
+
+  checker compare: #odd.
+
+  self
+    assert: ( checker checkAgainst: 10 );
```

Comment:
If the example is checking that they are numbers, shouldn't the check be against a number of another class, to prove that the implementation is working as expected? Something like `checkAgainst: 10.0`